packages/libs/src/sdk/scripts/generateServicesConfig.ts

```diff
@@ -44,9 +44,10 @@ const envConfigs: Record<'staging' | 'production', EnvironmentConfig> = {
     CLAIM_DISTRIBUTION_CONTRACT_ADDRESS:
       '0x74b89B916c97d50557E8F944F32662fE52Ce378d',
     ETH_OWNER_WALLET: '',
-    ETH_PROVIDER_URL: 'https://eth.staging.audius.co',
-    ETH_REGISTRY_ADDRESS: '0xF27A9c44d7d5DDdA29bC1eeaD94718EeAC1775e3',
-    ETH_TOKEN_ADDRESS: '0x5375BE4c52fA29b26077B0F15ee5254D779676A6',
+    ETH_PROVIDER_URL:
```

Comment:
don't think this should be public. should we have an `eth-sepolia.staging.audius.co` endpoint for now?

Reply:
removed, going to change eth.staging.audius.co when this merges